modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItTemporalIndexTest.java

```diff
@@ -158,6 +181,27 @@ public void testSearchGtLtWithPkIdxUsage(String table, String predicate, Object
         }
     }
 
+    /** Check range predicates with dynamic parameter. */
+    @ParameterizedTest(name = "table = {0}, predicate = {1}")
+    @MethodSource("geLeSearchDynParamArguments")
+    public void testSearchGtLtWithPkIdxUsageDynamicParam(String table, String predicate, Object parameter, Object result) {
+        assertQuery(format("SELECT /*+ FORCE_INDEX({}), DISABLE_RULE('TableScanToKeyValueGetRule') */ val FROM {} WHERE pk {} ORDER BY val",
```

Comment:
involved predicates can not match TableScanToKeyValueGetRule usage, thus it need to be removed if EQ predicates are not planned to use, otherwise it confused. Or probably it more clear to extend SORTED_INDEXES with "pkIndexName(table)" wdyt ?

Reply:
Thanks, Now in these tests I check all indexes in one loop (if I understood your suggestion correctly).